ui/src/app/collection/detail/manage-collection.component.spec.ts

```diff
@@ -273,7 +276,7 @@ describe("ManageCollectionComponent", () => {
 
       // Assert
       expect(actions).toBeTruthy()
-      expect(actions.length).toEqual(5)
+      expect(actions.length).toEqual(4)
```

Comment:
Looks like we need an explanation for this value of 4.  Perhaps name a constant for this magic number.  (I probably wrote this, so I should have done so before.  (facepalm))

Reply:
Previously the list of buttons was 5, now it is 4, since an if condition was imposed before filling the list.

a picture was added below for more detail

Reply:
To me, the best reference material is the source code, not the readme or a PR conversation. I think @hikr17 was asking for a comment next to the magic value, or a variable/constant in it's place that has a meaningful name.

Reply:
Yep, @JohnKallies.  That's what I was asking for.